.github/configs/.prettierignore

```diff
@@ -0,0 +1,4 @@
+# `release-please` doesn't generate prettier compliant output, see relevant issues:
+# https://github.com/googleapis/release-please/issues/1902
+# https://github.com/googleapis/release-please/issues/1802
+CHANGELOG.md
```

Comment:
Is this location where prettier will ignore it or release-please will ignore it? I'm surprised this shouldn't go into a config in `.trunk/`

Reply:
You're right! Valid configuration in `.trunk/` should satisfy this.